step-certificates/values.yaml

```diff
@@ -36,6 +36,7 @@ service:
   type: ClusterIP
   port: 443
   targetPort: 9000
+# nodePort: 32400
```

Comment:
Can you just set this to `""` instead of commenting on it? I always like to have all the values set in the values.yaml

Reply:
I've done it.